cfpb/capital-framework
# Fixes icon fonts and Avenir web font in docs

Adds missing variables to make Avenir web font work and icon font path load fonts correctly. Fixes #599 


## Changes

- missing font variables that used to be in `cf-theme-overrides.less` were added to `main.less`. Worth thinking about putting these in `cf-icons` and `cf-typography` where they are originally defined, but for now this is the quickest fix to getting CF docs showing the fonts correctly.

## Testing

- check out branch
- `npm start`
- Confirm icons and Avenir fonts work on [cf-typography](http://localhost:3000/components/cf-typography/) and [cf-icons](http://localhost:3000/components/cf-icons/) docs

## Screenshots


## Notes

-

## Todos

-

## Checklist


## Testing checklist

### Browsers

- [ ] Blackberry Bold

### Accessibility


### Other
